### ISSUE TITLE
tar_traceback()

### ISSUE THREAD

**Issue by wlandau** [wlandau/targets#217]
## Prework

* [ ] Read and agree to the [code of conduct](https://github.com/wlandau/targets/blob/main/CODE_OF_CONDUCT.md) and [contributing guidelines](https://github.com/wlandau/targets/blob/main/CONTRIBUTING.md).
* [x] If there is [already a relevant issue](https://github.com/wlandau/targets/issues), whether open or closed, comment on the existing thread instead of posting a new issue.
* [x] New features take time and effort to create, and they take even more effort to maintain. So if the purpose of the feature is to resolve a struggle you are encountering personally, please consider first posting a "trouble" or "other" issue so we can discuss your use case and search for existing solutions first.
* [x] Format your code according to the [tidyverse style guide](https://style.tidyverse.org/).

## Proposal

Should just load the workspace and return the traceback. This will be feasible after #214 because workspaces will be lighter.

**Comment by wlandau** [comment]
Implemented in #218.
